### DISCUSSION TITLE
How to filter counts by pixel size of objects

### DISCUSSION THREAD

**Discussion by BartVUacc** [Abe404/root_painter#135] [Q&A] [answered]
Hello,

I am using RootPainter to count enchytraeids in toxicity tests. I am already pretty happy with the model I have trained so far, but when I extract the count data for my dataset the counts are too high. When inspecting the annotated images I notice that a lot of noise (small pixels) are being counted. I see that in the SI of the paper some filter is applied based on pixel size, but so far I have been unable to replicate this. Any suggestions?

Kind regards,
Bart

**Comment by Abe404** [comment, marked as answer]
Hello Bart,

If you use 'extract region properties' which is available from the extras menu then you get a list (CSV file) of each region in the image. You can then process this list of regions in R, excel or python etc. The image name is in each row and also the size of each region. So some operation in R/excel/python can achieve the desired result. Are you familiar with any of these tools and able to use them to filter rows by value and count rows etc?

Kind regards,
Abraham

**Reply by BartVUacc** [reply]
Hi Abraham,

Thank you so much for your quick answer! I completely missed the "Extract region properties" option; now that I've tried it I think that will work wonderfully.

Thanks again,
Bart
